### PR TITLE
fix: avoid timeout due to long blake3 build from source

### PR DESCRIPTION
**Pull request by ddelange**

### CODE REVIEW

**Comment by ddelange** [comment]
cc @maartenbreddels saw that master was still flaky

File: .github/workflows/wheel.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ concurrency:
 
 jobs:
   build_wheels_matrix:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```
Comment (ddelange) [inline .github/workflows/wheel.yml:24]: pinning OS versions in `wheel.yml` hoping to get less unexpected failues in CI (due to edits to the runner image like removing libmagic)

**Comment by ddelange** [comment]
@maartenbreddels ready for merge & release

**Comment by maartenbreddels** [comment]
Awesome. Will check if we can do a release now.

**Pull request merged**